src/generic/FormSwitchGroup.jsx

```diff
@@ -27,7 +27,7 @@ export default function FormSwitchGroup({
             {helpText}
           </Form.Text>
         </div>
-        <Form.Switch
+        <SwitchControl
```

Comment:
Use `SwitchContorl` for unlabeled control
https://paragon-edx.netlify.app/components/form/form-switch/